cmd/build/helmify/replacements.go

```diff
@@ -114,6 +114,80 @@ var replacements = map[string]string{
 
 	"- HELMSUBST_MUTATION_STATUS_ENABLED_ARG": `{{ if not .Values.disableMutation}}- --operation=mutation-status{{- end }}`,
 
+	"- HELMSUBST_CONTROLLER_MANAGER_OPERATIONS": `
+        {{- if not (hasKey .Values.controllerManager "disableWebhookOperation") }}
+        - --operation=webhook
+        {{- else if not .Values.controllerManager.disableWebhookOperation }}
+        - --operation=webhook
+        {{- end }}
+        {{- if hasKey .Values.controllerManager "disableGenerateOperation" }}
```

Comment:
This check may not be necessary if we supply defaults in values.yaml - applied to all similar places.